elastic/integrations
# [Uptime] Feature/synthetics integration package

Please label this PR with one of the following labels, depending on the scope of your change:
- Enhancement

## What does this PR do?

This PR adds the spec for the Elastic Synthetics integration package, which includes support for HTTP, ICMP, and TCP monitors.

## Checklist

- [ ] I have verified that all data streams collect metrics or logs.  (Collects synthetics)

## Author's Checklist


- [ ] Ensure that field data types are compatible with heartbeat config

## How to test this PR locally

1. Install Elastic Package (https://github.com/elastic/elastic-package). 
2. Navigate to `packages/synthetics` in the integration repo
3. Run `elastic-package build` to build the synthetics package
4. Run `elastic-package stack up --version 8.0.0-SNAPSHOT` to serve kibana, es, and package registry
5. Log into kibana and navigate to Management -> Fleet
6. Once in fleet, navigate to the Integrations tab and search for Synthetics
7. Click Add Integration once on the Synthetics page to test. You should see the default form builder UI with all our configuration. 

## Related issues
Relates elastic/kibana#91062
